modules/apps/collaboration/social/social-group-statistics-web/src/main/resources/META-INF/resources/add_activity_counter.jsp

```diff
@@ -64,7 +64,7 @@ Collections.sort(activityCounterNames, new SocialActivityCounterNameComparator(l
 		<liferay-ui:message key="group-statistics-add-counter-first-text" />
 	</span>
 
-	<aui:select inlineField="<%= true %>" label="" name='<%= portletResourceNamespace + "preferences--displayActivityCounterName" + index + "--" %>' title="display-activity-counter-name" useNamespace="<%= false %>">
+	<aui:select inlineField="<%= true %>" label="" name='<%= portletResourceNamespace + "displayActivityCounterName" + index %>' title="display-activity-counter-name" useNamespace="<%= false %>">
```

Comment:
Hey @gregory-bretall 

I do not fully understand why do we need this change. 

What happens if the customer has stored a preference like "preferences--displayActivityCounterName-" and your changes are appliedv (I mean, deleting the "preferences--" part of the property) . Will the system retrieve the configured value or will return an empty/or default value??